lib/omnibus/packagers/rpm.rb

```diff
@@ -830,5 +833,40 @@ def remove(packages)
         shellout!("yum -y remove #{packages}")
       end
     end
+
+    def compression_level(val = nil)
+      unless val.nil?
+        unless val >= 0 && val <= 9
+          raise InvalidValue.new(:compression_level, 'be an Integer between 0 and 9 included')
+        end
+      end
+      @compression_level = val || 6
+    end
+    expose :compression_level
+
+    def compression_threads(val = nil)
+      unless val.nil?
+        unless val > 0 && val < 32
+          raise InvalidValue.new(:compression_threads, 'be a stricly positive and lower than 32 Integer')
+        end
+      end
+      @compression_threads || 1
+    end
+    expose :compression_threads
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏‏
Maybe we could share this between `rpm` and `deb` to avoid duplication.
Feel free to ignore this one

Reply:
I'm not sure how to proceed with that, although I agree on the principle.
We could move some properties to the base packager class, but this would allow the properties to be set on packagers that actually don't support them.

Since it's only about 2 properties (`compression_algo` needs the RPM specific translation from `xz` to `xzdio` and similar) I'd tend to leave that be for now (and hope that we get rid of omnibus before code duplication becomes too important :crossed_fingers: )